xml/System/String.xml

```diff
@@ -11456,9 +11456,10 @@ Each element of `separator` defines a separate delimiter that consists of a sing
 5. <xref:System.String.Empty?displayProperty=nameWithType>, which represents the empty string that follows the "_" character at index 5.
 
 ### The separator array
- If the `separator` parameter is `null` or contains no characters, white-space characters are assumed to be the delimiters. White-space characters are defined by the Unicode standard and the <xref:System.Char.IsWhiteSpace%2A?displayProperty=nameWithType> method returns `true` if they are passed to it.
 
- If the `separator` parameter is `null`, compiler overload resolution fails. To unambiguously identify the called method, your code must indicate the type of the `null`. The following example shows several ways to unambiguously identify this overload.
+If the `separator` parameter is `null` or contains no characters, white-space characters are assumed to be the delimiters. White-space characters are defined by the Unicode standard and the <xref:System.Char.IsWhiteSpace%2A?displayProperty=nameWithType> method returns `true` if they are passed to it.
+
+If the `separator` parameter is `null`, compiler overload resolution fails. To unambiguously identify the called method, your code must indicate the type of the `null`. The following example shows several ways to unambiguously identify this overload.
```

Comment:
This seems to contradict the previous paragraph, which says if it's null, whitespace is assumed.